lib/src/components/NxProgressBar/__tests__/NxProgressBar.test.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+
+import NxProgressBar from '../NxProgressBar';
+
+import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
+
+describe.only('NxProgressBar', function() {
+  const getShallowComponent = enzymeUtils.getShallowComponent(NxProgressBar, { value: 50 });
+
+  it('sets the correct max and value to the progress element',
+      function() {
+        const progressSelector = 'progress.nx-progress-bar__progress';
+        expect(getShallowComponent({ value: 0 }).find(progressSelector)).toHaveProp('max', 100);
+        expect(getShallowComponent({ value: 0, max: 50 }).find(progressSelector)).toHaveProp('max', 50);
+        expect(getShallowComponent({ value: 20, max: 40 }).find(progressSelector)).toHaveProp('value', 20);
+      }
+  );
+
+  it('displays the correct className and hides label when inlineCounter is set to true',
+      function() {
+        const component = getShallowComponent({ value: 100, label: 'hello' });
+        const inlineCounterComponent = getShallowComponent({ value: 100, inlineCounter: true, label: 'hello' });
+
+        const labelTextSelector = '.nx-progress-bar__label-text';
+
+        expect(component).not.toHaveClassName('nx-progress-bar--inline-counter');
+        expect(component.find(labelTextSelector)).toExist();
+        expect(component.find(labelTextSelector)).toHaveText('hello');
+
+        expect(inlineCounterComponent).toHaveClassName('nx-progress-bar--inline-counter');
+        expect(inlineCounterComponent.find(labelTextSelector)).not.toExist();
+      }
+  );
+
+  it('sets the correct counter percentage value',
+      function() {
+        const counterSelector = '.nx-counter';
+
+        expect(getShallowComponent({ value: 20, max: 40 }).find(counterSelector)).toHaveText('50%');
+        expect(getShallowComponent({ value: 0 }).find(counterSelector)).toHaveText('0%');
+        expect(getShallowComponent({ value: 50 }).find(counterSelector)).toHaveText('50%');
+        expect(getShallowComponent({ value: 100 }).find(counterSelector)).toHaveText('100%');
+      }
+  );
+
+  it('sets the correct success className when progress is 100%',
+      function() {
+        const component = getShallowComponent({ value: 0 });
+        const successComponent = getShallowComponent({ value: 100 });
```

Comment:
Probably should test this with a custom max as well

Reply:
> Probably should test this with a custom max as well

Fixed: 40650085762ac5ec62e6e5927255bc82dc399347